maitake/README.md

```diff
@@ -0,0 +1,63 @@
+# maitake
+
+["Dancing mushroom"][maitake-wiki] &mdash; an async runtime construction kit.
+
+[![crates.io][crates-badge]][crates-url]
+[![Documentation][docs-badge]][docs-url]
+[![Documentation (HEAD)][docs-main-badge]][docs-main-url]
+[![MIT licensed][mit-badge]][mit-url]
+[![Test Status][tests-badge]][tests-url]
+[![Loom Models][loom-badge]][loom-url]
+[![Sponsor @hawkw on GitHub Sponsors][sponsor-badge]][sponsor-url]
+
+[crates-badge]: https://img.shields.io/crates/v/maitake.svg
+[crates-url]: https://crates.io/crates/maitake
+[docs-badge]: https://docs.rs/maitake/badge.svg
+[docs-url]: https://docs.rs/maitake
+[docs-main-badge]: https://img.shields.io/netlify/3ec00bb5-251a-4f83-ac7f-3799d95db0e6?label=docs%20%28main%20branch%29
+[docs-main-url]: https://mycelium.elizas.website/maitake
+[mit-badge]: https://img.shields.io/badge/license-MIT-blue.svg
+[mit-url]: ../LICENSE
+[tests-badge]: https://github.com/hawkw/mycelium/actions/workflows/ci.yml/badge.svg?branch=main
+[tests-url]: https://github.com/hawkw/mycelium/actions/workflows/ci.yml
+[loom-badge]: https://github.com/hawkw/mycelium/actions/workflows/maitake.yml/badge.svg?branch=main
+[loom-url]: https://github.com/hawkw/mycelium/actions/workflows/maitake.yml
+[sponsor-badge]: https://img.shields.io/badge/sponsor-%F0%9F%A4%8D-ff69b4
+[sponsor-url]: https://github.com/sponsors/hawkw
+[maitake-wiki]: https://en.wikipedia.org/wiki/Grifola_frondosa
+
+## what is it?
+
+This library is a collection of modular components for building a Rust
+async runtime based on [`core::task`] and [`core::future`], with a focus on
+supporting `#![no_std]` projects.
+
+Unlike other async runtime implementations, `maitake` does *not* provide a
```

Comment:
Compare to `smol` perhaps?

Reply:
oh good point, smol is in _some_ ways similar, but the primitives it exposes are somewhat higher-level

Reply:
Gotcha, you know better than me! I just knew that `smol` had been re-used in multiple executors as sort of a "core" building block.

Reply:
i might add a whole "comparison to other crates" section in a follow-up PR?

Reply:
Sounds good! The natural comparisons would be embassy (also no_std focused, but different), smol (as another building-block-ish crate), and tokio/async-std (as full-featured, but std-focused runtimes/ecosystems)

Reply:
> Gotcha, you know better than me! I just knew that `smol` had been re-used in multiple executors as sort of a "core" building block.

yeah, it's still a slightly higher level primitive, i think --- this crate is more comparable to something like [`async-task`](https://docs.rs/async-task/latest/async_task/) which is a dependency of `smol`.

Reply:
Specifically for embassy, maybe focus on the fact that maitake is optionally-alloc, and supports dynamic task spawning